.devcontainer/devcontainer.json

```diff
@@ -11,32 +10,23 @@
         // Configure properties specific to VS Code.
         "vscode": {
             "settings": {
-                "python.defaultInterpreterPath": "/usr/local/bin/python",
                 "python.linting.enabled": true,
-                "python.linting.pylintEnabled": true,
                 "python.formatting.provider": "yapf",
-                "python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
-                "python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
-                "python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
-                "python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
-                "python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
-                "python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
-                "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint"
+                "python.formatting.yapfPath": "yapf"
             },
             "extensions": [
                 "ms-python.python",
                 "ms-python.vscode-pylance",
-                "ms-azuretools.vscode-docker"
+                "ms-azuretools.vscode-docker",
+                "donjayamanne.githistory"
             ]
         }
     },
     // Run the install script on create
-    "postCreateCommand": "pip install .[rllib] .[pettingzoo]",
-    // Vscode only user fix - see https://aka.ms/vscode-remote/containers/non-root.
-    "remoteUser": "vscode",
+    "postCreateCommand": "pip install .[rllib]",
```

Comment:
Why drop `.[pettingzoo]`?

Reply:
The pettingzoo example in meltingpot is broken at the moment, so I'm not sure it makes sense to install this. On balance however it probably makes sense to leave in for now, so I'll switch it back

```suggestion
    "postCreateCommand": "pip install .[rllib] .[pettingzoo]",
```

I'll create a separate issue for the current bug.